docker-compose.yml.example

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  runner:
+    image: media-uncovered/newsanalysis:acceptance
+    build: .
+    environment:
+      DB_HOST: "DB_HOST"
+      DB_NAME: "DB_NAME"
+      DB_PORT: "DB_PORT"
+      DB_USER: "DB_USER"
+      DB_PASSWORD: "DB_PASSWORD"
+      NO_DOCS: "NO_DOCS"
```

Comment:
Does specifying the NO_DOCS in the docker-compose file requires a rebuild for every new value?
If so, I would suggest to remove it from here and add it as an input parameter to the run.py file